slack.py

```diff
@@ -31,30 +32,36 @@ def send_messages(projects_by_owner):
     for owner in projects_by_owner.keys():
         slack_user = USERS_MAP.get(owner, owner)
         message = "Hey @{}, I've noticed you are one of the owners of the following projects:\n".format(slack_user)
+        send_message_to_this_owner = False
         for project in projects_by_owner.get(owner):
             project_id = project.get('projectId')
             org = ORGS_NAME_MAPPING.get(project.get('parent').get('id'))
             created_days_ago = int(project.get('createdDaysAgo'))
             cost = project.get('costSincePreviousMonth', 0.0)
             currency = project.get('costCurrency', '$')
             emoji = ''
-            if cost > COST_ALERT_THRESHOLD:
-                emoji = ' ' + COST_ALERT_EMOJI
-            message += "> - `{}/{}` created `{} days ago`, costing *`{}`* {}.{}\n".format(org, project_id, created_days_ago, cost, currency, emoji)
+            if cost <= COST_MIN_TO_NOTIFY:
+                logger.debug('- `{}/{}`, will not be in the message, due to its cost being lower than the minimum warning value'.format(owner, project_id))
+            else:
+                if cost > COST_ALERT_THRESHOLD:
+                    emoji = ' ' + COST_ALERT_EMOJI
+                    send_message_to_this_owner = True
```

Comment:
Same issue here (see chat channel).

Reply:
these changes are made in commit 9f13dbc